lib/check_game.rb

```diff
@@ -0,0 +1,16 @@
+class GameCheck
+  def vallidate_position(data)
+    pattern = /[0-8]/
+    data.length == 1 && data.match?(pattern) ? true : false
+  end
```

Comment:
vallidate_position has been moved to the GameBoard class. We are not interested in merging both methods because they are called at different times to perform different functions.